dumper/lib-ext-hive-metastore/gradle.lockfile

```diff
@@ -24,7 +24,7 @@ com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.12.0=hive312RuntimeCla
 com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.12.0=hive312RuntimeClasspath
 com.fasterxml.jackson:jackson-bom:2.12.0=hive312RuntimeClasspath
 com.fasterxml.jackson:jackson-bom:2.18.1=compileClasspath,runtimeClasspath,testCompileClasspath,testFixturesRuntimeClasspath,testRuntimeClasspath
-com.fasterxml.woodstox:woodstox-core:5.0.3=hive312RuntimeClasspath
+com.fasterxml.woodstox:woodstox-core:5.4.0=hive312RuntimeClasspath
```

Comment:
I suggest to add a more descriptive PR title e.g. "Update woodstox-core library to version 5.4.0". This should be easy to identify changes by looking at PRs and commits history. "update lockfile" doesn't deliver any useful information.